app/src/main/java/com/example/periodtracker/MainActivity.java

```diff
@@ -2,13 +2,81 @@
 
 import androidx.appcompat.app.AppCompatActivity;
 
+import android.content.Intent;
 import android.os.Bundle;
+import android.view.View;
+import android.view.animation.AlphaAnimation;
+import android.widget.Button;
+import android.widget.ImageView;
+import android.widget.TextView;
+import com.google.android.material.bottomnavigation.BottomNavigationView;
+import java.util.Calendar;
+import java.util.Locale;
 
 public class MainActivity extends AppCompatActivity {
 
+    private TextView displayDay;
+    private TextView displayMonth;
+    private Calendar currentDate;
+    private Locale locale;
+    private AlphaAnimation buttonClickAnimation;
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+
+        buttonClickAnimation = new AlphaAnimation(1,0);
+        buttonClickAnimation.setDuration(300);
+        currentDate = Calendar.getInstance();
+        locale = Locale.getDefault();
+        displayDay = findViewById(R.id.day);
+        displayMonth = findViewById(R.id.month);
+        displayDay.setText(String.valueOf(currentDate.get(Calendar.DAY_OF_MONTH)));
+        displayMonth.setText(currentDate.getDisplayName(Calendar.MONTH, Calendar.SHORT, locale));
+
+        Button btn_tick = findViewById(R.id.btn_tick);
+        btn_tick.setOnClickListener(v -> {
```

Comment:
Declare OnClick functions as separate functions like something like:
`public void onSymptomsClick(View view) { //code };`
These functions can be set for buttons in the appropriate `layout.xml` file under attributes for that button. This will just help keep the code clean and easier to maintain

Reply:
![image](https://github.com/COMP-3013-Team-5-2023-2024/TEAM-5-COMP-3018/assets/148252255/01894fe6-4724-49f2-af05-5b8bf5f3a919)
When trying to do it, the ide recommends me continuing to use setOnClickListener, after searching a bit, leaving it with setOnClickListener may be better..? https://stackoverflow.com/questions/22172448/button-setonclicklistener-vs-androidonclick#:~:text=XML%20onClick%20is%20good%20for,do%20the%20exact%20same%20thing.